### DISCUSSION TITLE
Saved Withdrawal Amounts

### DISCUSSION THREAD

**Discussion by MBanner97** [runelite/runelite#15536] [Q&A] [answered]
Would like to be able to right click and have a saved withdrawal amount for an item from my bank. Is this possible?

**Comment by Enriath** [comment]
You're going to have to be more specific about what you mean.

**Comment by MBanner97** [comment]
Hey there,

I’d love to be able to right click on an item in my bank,right click on the
item and automatically have a saved set number of what I want to withdraw
at all times.

**Reply by Enriath** [reply]
That's what the `Withdraw X` feature of the game is for. Set a value for that and it'll remain until you change it.

Setting multiple different X values is not something that's possible.

**Reply by Talados** [reply]
Maybe they meant saving a different amount for different items?

**Reply by Enriath** [reply]
> Setting multiple different X values is not something that's possible.

**Comment by Enriath** [comment, marked as answer]
That's what the `Withdraw X` feature of the game is for. Set a value for that and it'll remain until you change it.

Setting multiple different X values is not something that's possible.
